job_creator/test/test_utils.py

```diff
@@ -52,3 +58,108 @@ def test_ensure_ceph_path_exists(self):
 
         self.assertEqual(end_path, "/tmp/ceph/mari/RBNumber/unknown/autoreduced")
         os.removedirs("/tmp/ceph/mari/RBNumber/unknown/autoreduced")
+
+    def test_extract_useful_parts_from_image_with_https(self):
```

Comment:
Could you parameterise some of these tests?

Reply:
Sorted!